src/input.ts

```diff
@@ -8,7 +9,8 @@ interface EnvironmentVariable {
 
 class Input {
   readonly options = {
-    envRegex: /^(?:RENOVATE_\w+|LOG_LEVEL|GITHUB_COM_TOKEN|NODE_OPTIONS|(?:HTTPS?|NO)_PROXY|(?:https?|no)_proxy)$/,
+    envRegex:
+      /^(?:RENOVATE_\w+|LOG_LEVEL|GITHUB_COM_TOKEN|NODE_OPTIONS|(?:HTTPS?|NO)_PROXY|(?:https?|no)_proxy)$/,
```

Comment:
revert

Reply:
this was automatic made by the linting during husky automatic check on the commit, if you want I can revert and avoiding the linting. Is just a newline.

Reply:
I had the exact same issue with a brand new clone.

Reply:
It happens due to the `.eslintrc.js` default preset extended here https://github.com/renovatebot/github-action/blob/main/.eslintrc.js#L32 that defines a max line length of 80 chars (https://eslint.org/docs/latest/rules/max-len#options). Is automatically reformatted by husky during the pre-commit stage with the `yarn lint-staged` command defined [here](https://github.com/renovatebot/github-action/blob/main/.husky/pre-commit#L4). Anyway I'm reverting the change and committing it by skipping husky pre-commit stage 👍

Reply:
fixed in main